test/e2e/apps/disruption.go

```diff
@@ -148,7 +148,7 @@ var _ = SIGDescribe("DisruptionController", func() {
 		if c.shouldDeny {
 			expectation = "should not allow an eviction"
 		}
-		ginkgo.It(fmt.Sprintf("evictions: %s => %s", c.description, expectation), func() {
+		ginkgo.It(fmt.Sprintf("evictions: %s => %s [Serial]", c.description, expectation), func() {
```

Comment:
we only need to do this for the exclusive cases, right? those appear to be the only ones flaking regularly (https://storage.googleapis.com/k8s-gubernator/triage/index.html?date=2020-02-09&ci=0&pr=1&test=DisruptionController)

Reply:
Yeah, I think you are right. I will update the PR.